app/models/recipe.rb

```diff
@@ -1,3 +1,7 @@
 class Recipe < ApplicationRecord
   belongs_to :user
+  has_many :recipe_ingredients
+  has_many :dietary_information_recipes
+
+  validates_presence_of :total_time, :title, :serving, :instruction
```

Comment:
Should validate the presense of ingredients too :)

Reply:
ah, that is true. i will fix that and have to check if that is working, since it is an association.

Reply:
@akramerin still no recipe ingredients in the validate presence here :)